aspnetcore/web-api/action-return-types.md

```diff
@@ -156,7 +156,7 @@ The `HttpResults` can be useful when sharing code between Minimal APIs and Web A
 
 The <xref:Microsoft.AspNetCore.Http.HttpResults> namespace contains classes that implement the <xref:Microsoft.AspNetCore.Http.IResult> interface. The `IResult` interface defines a contract that represents the result of an HTTP endpoint. The static [Results](<xref:Microsoft.AspNetCore.Http.Results>) class is used to create varying `IResult` objects that represent different types of responses.
 
-The [Built-in results](/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-7.0&preserve-view=true#bimr7) table shows the common result helpers.
+The [Built-in results](/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-7.0&preserve-view=true#built-in-results) table shows the common result helpers.
```

Comment:
custom html links are preferred because they don't break with each version upgrade. What's wrong with `#bimr7` ?

Reply:
감사합니다

Reply:
Thanks. Sorry to bother.  I assumed there was a reason for the `#bimr7` fragment;  however in this case it just goes to the top of a long page, rather than directly to the table being referenced.  Is there some magic that should translate the `#bimr7` fragment to the `id="built-in-results"` on the destination page?

Reply:
It's a standard HTML in page anchor, we should figure out why it's not working. Perhaps there's a duplicate on the page. Use the F12 tools to see what's generated.

Reply:
Can you provide a link to `/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-7.0&preserve-view=true#built-in-results`
I don't even see that H2.

Reply:
Looks to be an H3.

<img width="1037" alt="Screen Shot 2023-01-06 at 11 26 55 AM" src="https://user-images.githubusercontent.com/18443511/211054594-4d2f13b4-56b4-40ce-90c1-61f271db4a9a.png">

Reply:
https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-7.0&preserve-view=true#built-in-results